CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - API 21, 22 support
 
 ### Breaking Changes
+- Dropped Context from MeetingOptions, added to CallComposite.launch(..)
```

Comment:
Lets have two separate records for these changes in this format:
-Remove Context from Class.Method(constructor)
-Add required parameter to Class.Method

Reply:
Updated docs.